src/themes/teams/components/Text/textVariables.ts

```diff
@@ -7,41 +6,15 @@ export interface TeamsTextVariables extends TextVariables {
   timestampHoverColor: string
 }
 
-export default (siteVariables): TeamsTextVariables => {
-  const colorVariant = 500
-
-  return {
-    colors: mapColorsToScheme(siteVariables, colorVariant),
-    atMentionOtherColor: siteVariables.brand06,
-    atMentionMeColor: siteVariables.orange04,
-    atMentionMeFontWeight: siteVariables.fontWeightBold,
-    disabledColor: siteVariables.gray06,
-    errorColor: siteVariables.red,
-    importantWeight: siteVariables.fontWeightBold,
-    importantColor: siteVariables.red,
-    successColor: siteVariables.green04,
-    timestampColor: siteVariables.gray04,
-    timestampHoverColor: siteVariables.gray02,
-
-    fontSizeExtraSmall: siteVariables.fontSizes.smaller,
-    fontLineHeightExtraSmall: siteVariables.lineHeightExtraSmall,
-
-    fontSizeSmall: siteVariables.fontSizes.small,
-    fontLineHeightSmall: siteVariables.lineHeightSmall,
-
-    fontSizeMedium: siteVariables.fontSizes.medium,
-    fontLineHeightMedium: siteVariables.lineHeightBase,
-
-    fontSizeLarge: siteVariables.fontSizes.large,
-    fontLineHeightLarge: siteVariables.lineHeightSmall,
-
-    fontSizeExtraLarge: siteVariables.fontSizes.larger,
-    fontLineHeightExtraLarge: siteVariables.lineHeightSmall,
-
-    fontWeightLight: siteVariables.fontWeightLight,
-    fontWeightSemilight: siteVariables.fontWeightSemilight,
-    fontWeightRegular: siteVariables.fontWeightRegular,
-    fontWeightSemibold: siteVariables.fontWeightSemibold,
-    fontWeightBold: siteVariables.fontWeightBold,
-  }
-}
+export default (siteVariables): Partial<TeamsTextVariables> => ({
+  atMentionOtherColor: siteVariables.brand06,
+  atMentionMeColor: siteVariables.orange04,
+  atMentionMeFontWeight: siteVariables.fontWeightBold,
+  disabledColor: siteVariables.gray06,
+  errorColor: siteVariables.red,
+  importantWeight: siteVariables.fontWeightBold,
+  importantColor: siteVariables.red,
+  successColor: siteVariables.green04,
+  timestampColor: siteVariables.gray04,
+  timestampHoverColor: siteVariables.gray02,
```

Comment:
Not for this PR, but maybe now that we have the color palette we should redefine the colors we are using with using variants with number, as it is done for the colors in the base theme.